src/objects/npo02__Households_Settings__c.object

```diff
@@ -2,6 +2,18 @@
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
     <enableEnhancedLookup>false</enableEnhancedLookup>
     <fields>
+        <fullName>Field_for_Effective_Exchange_Rate__c</fullName>
```

Comment:
@smarovich FYI - New custom settings field and related inline help text. These PR is being merged into the larger roll-ups branch so it won't appear in the release notes for quite a while.

Reply:
Hi @force2b , I'd make some minor changes to the Help text:

The API name of a field, on both the OppPayment__c and Allocation__c objects, where the exchange rate used for conversion is stored. The referenced field must be a Number field with a minimum of 6 decimals. This field is only used if Advanced Currency Management is enabled.

Also, for the field name, would "Exchange Rate for Conversion" work?

Reply:
Thanks for the quick review. Two things:
* The help text is limited to 255 characters. I can make that fit by shortening some of the terms (such as OppPayment__c to just Payment and Management to just mgt).
* I'm hesitant on describing it as "exchange rate for conversion". The field is for their use. It may be used for conversion, it may be used for something else (though conversion is a likely use case). It will contain the exchange rate, but how they use it is sort of up to the customer. I hope that makes sense.

Reply:
Thanks, Mike. How about this for the help text: 
The API name of a field, on both the OppPayment__c and Allocation__c objects, where the exchange rate is stored. It must be a Number field with a minimum of 6 decimals. This field is used only if Advanced Currency Management is enabled.

As for the field name, could we keep it simple and call it "Exchange Rate" or "Exchange Rate Field"?

Reply:
Thanks @smarovich. Two thoughts:
* What do you think about a label of "Custom Exchange Rate Field"?
* For the inline help text, how about this small change: "... where the exchange rate is *optionally* stored. ... "

Reply:
Your suggestions sound good to me, Mike! Thanks!

Reply:
Great. Thank you.